src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => {
+          const isAction = tab === selectedTab;
```

Comment:
- What action is it? `isTabSelected` or `isTabActive` I think describes better this variable
- WE suggest you never compare the object with another object with strict comparison. It could produce a lot of bugs. Compare object using their unique identifier (id, title)
![image](https://github.com/mate-academy/react_tabs-js/assets/63243136/cd06de7d-66b6-4630-b5f8-4f8ea6c87ed1)